packages/clima_ui/lib/widgets/forecast_widget.dart

```diff
@@ -38,46 +38,55 @@ class ForecastHorizontal extends HookWidget {
         final forecast = forecasts.forecasts[index];
         return Padding(
           padding: EdgeInsets.symmetric(
-            horizontal: 8,
-            vertical: () {
-              if (MediaQuery.of(context).size.height <= 650) {
-                return 4.0;
-              }
-              return 8.0;
-            }(),
-          ),
-          child: Center(
-            child: Column(
-              mainAxisAlignment: MainAxisAlignment.spaceBetween,
-              children: <Widget>[
-                Text(
-                  DateFormat('E, h a').format(
-                    forecast.date.toUtc().add(forecast.timeZoneOffset),
-                  ),
-                  style: TextStyle(
-                    color: Theme.of(context)
-                        .textTheme
-                        .subtitle1
-                        .color
-                        .withAlpha(130),
-                  ),
+              horizontal: 8,
+              vertical: () {
+                if (MediaQuery.of(context).size.aspectRatio == 9 / 16) {
```

Comment:
Did you mean `<=`, or is this on purpose? If it is, please add a comment to the code clarifying this.

Reply:
It's on purpose.